system-tests/tests/buildiso-@

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Check that Cobbler is able to build customized ISOs by the "cobbler buildiso" command with the addition of the
+# airgapped flag
+
+source ${SYSTESTS_PRELUDE} && prepare
+
+build_iso_test=${TEST_NAME#buildiso-}
+
+trap cleanup EXIT
```

Comment:
You can just unmount after the mount, no? The mounted ISO is only required for `cobbler import`.

Reply:
Well we do copy files for `cobbler buildiso`, so I believe this is not possible. Or am I wrong?

Reply:
I was under the impression that this works in the following manner:

```
# mount -o loop, ro ${iso} ${mp}
# cobbler import --name leap --path ${mp}
# umount ${mp}
```

Nothing else touches ${mp} after `cobbler import`.

Reply:
Btw since you are calling cobbler import before `set -e`, you should do `cobbler import ... || exit 1` in order to catch the failure.

Reply:
Well Cobbler touches it on every restart of the service AND for a `cobbler buildiso` we copy the whole content of the disk again into the ISO. Also after an `cobbler import` the kernel and initrd are synced from the `${mp}` on a `cobbler sync`.

Reply:
> Btw since you are calling cobbler import before set -e, you should do cobbler import ... || exit 1 in order to catch the failure.

I will fix it with a force push again.

Reply:
Oh... well yeah, then trap is a good way to do it.